src/main/resources/schema/query.graphqls

```diff
@@ -1,4 +1,17 @@
 type Query {
+  "Get a wish by ID"
   wish(id: ID): Wish
-  allWish: [Wish!]!
+
+  "Gets all Wishes"
+  allWishes: [Wish!]!
```

Comment:
Q: What if there are no wishes?
recomendation: allWishes: [Wish]!
This will return an empty array if there are no wishes

Reply:
@helipilot50 As far as I know [Wish!]! would also allow to return an empty array - just none of it's elements are allowed to be null. [Wish]! would allow null elements in the array.